core/src/main/scala/org/apache/spark/sql/delta/commands/RestoreTableCommand.scala

```diff
@@ -52,6 +54,8 @@ import org.apache.spark.util.SerializableConfiguration
  * 7) If table was modified in parallel then ignore restore and raise exception.
  *
  */
+
+@DeveloperApi
```

Comment:
Do we need this tag?

Reply:
I just wanted to notify delta users that it not good idea to use this class directly. But after reviewing delta code I found that this annotation is not popular in delta project, so I'll remove it. Also, according to  Scala AI doc only io.delta.tables is a public API so users should not instantiate this class manually.